api/maven-api-model/src/main/mdo/maven.mdo

```diff
@@ -121,34 +121,39 @@
           <version>3.0.0+</version>
           <required>true</required>
           <description>
-            A universally unique identifier for a project. It is normal to
+            Returns a string that defines a collection of related projects. It is recommended to
             use a fully-qualified package name to distinguish it from other
-            projects with a similar name (eg. {@code org.apache.maven}).
+            projects with a similar name (for example, {@code org.apache.maven}).
+            This form is now required to create a new group ID in Maven Central.
+            However, older projects did not always follow this convention,
+            and often use a single word such as jaxen or jdom as the group ID.
           </description>
           <type>String</type>
         </field>
         <field>
           <name>artifactId</name>
           <version>3.0.0+</version>
           <required>true</required>
-          <description>The identifier for this artifact that is unique within the group given by the
+          <description>Returns a string that defines a collection of versions within the group given by the
```

Comment:
It sounds weird to define the artifactId as a collection of version.  That's not really what is it imho.  The `groupId:artifact` represents an artifact, and several versions of it can be available.  But an `artifatId` is _not_ a collection of version.

Reply:
Yes, this is tricky. There might be a better way of putting this. Note that I am not saying here that a groupId:artifactId *is* a collection of versions but that it *defines* a collection of versions. Perhaps I could change it to *identifies* a collection of versions. I could also make a case for *delineates* a collection of versions, though that word is a little obscure for technical writing.